Plugins/Creature/Creature.cpp

```diff
@@ -1876,4 +1882,38 @@ ArgumentStack Creature::SetDisarmable(ArgumentStack&& args)
     }
     return Services::Events::Arguments();
 }
+
+ArgumentStack Creature::SetFaction(ArgumentStack&& args)
+{
+    int32_t retVal = -1;
+    if (auto *pCreature = creature(args)){
+        const auto nFacId = Services::Events::ExtractArgument<int32_t>(args);
+        auto* pFaction= Globals::AppManager()->m_pServerExoApp->m_pcExoAppInternal->m_pFactionManager->GetFaction(nFacId);
+        if (pFaction){
+            pFaction->AddMember(pCreature->m_idSelf);
+            retVal = 1;
+        }else{
+            LOG_NOTICE("NWNX_Faction function called with invalid faction id");
```

Comment:
This is no longer a `NWNX_Faction` function. But you probably don't need the print anyway.
Same for other two prints.

Reply:
Corrected log message